src/test/java/com/lambdatest/JavaSmartUISDK

```diff
@@ -0,0 +1,86 @@
+package com.lambdatest;
+
+import java.lang.reflect.Method;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.*;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testng.Assert;
+import org.testng.ITestContext;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import io.github.lambdatest.*;
+
+public class JavaSmartUISDK {
+
+    private RemoteWebDriver driver;
+    private String Status = "failed";
+    private String githubURL = System.getenv("GITHUB_URL");
+
+
+    @BeforeMethod
+    public void setup(Method m, ITestContext ctx) throws MalformedURLException {
+        String username = System.getenv("LT_USERNAME") == null ? "Your LT Username" : System.getenv("LT_USERNAME");
+        String authkey = System.getenv("LT_ACCESS_KEY") == null ? "Your LT AccessKey" : System.getenv("LT_ACCESS_KEY");
+        String hub = "@hub.lambdatest.com/wd/hub";
```

Comment:
For SDK we need 2 samples, one with hub and one without hub

Reply:
Yes, I am making one for the local connection today